Application/EdFi.Ods.SandboxAdmin/Views/Shared/Layout.cshtml

```diff
@@ -37,14 +37,18 @@
 <head>
     <title>@ViewBag.Title</title>
     @RenderSection("head", required: false)
-    <link rel="stylesheet" href="~/css/site.min.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.0.0/css/flatly/bootstrap.min.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.0.3/css/font-awesome.css" />
+    <link rel="stylesheet" href="~/css/site.css" />
     <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/knockout/3.3.0/knockout-min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/knockout.mapping/2.4.1/knockout.mapping.min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.4/js/bootstrap.min.js"></script>
 </head>
 <body data-auth="@isAuthenticated.ToString()">
-    <script src="~/js/bundles.min.js"></script>
+    <script src="~/js/extensions.js"></script>
+    <script src="~/js/globals.js"></script>
+    <script src="~/js/edfiadmin.auth.js"></script>
```

Comment:
I missed this in the acceptance criteria but underscore was also a part of the bundles.min.js